autosklearn/ensemble_builder.py

```diff
@@ -71,7 +72,13 @@ def __init__(
                 If performance_range_threshold > 0, might return less models
             max_models_on_disc: int
                Defines the maximum number of models that are kept in the disc.
-               If int, it must be greater or equal than 1. If None, feature is disabled.
+               If int, it must be greater or equal than 1, and dictates the max number of
+               models to keep.
+               If float, it will be interpreted as the max megabytes allowed of disc space. That
+               is, if the number of ensemble candidates require more disc space than this float
+               value, the worst models will be deleted to keep within this budget.
+               models will be deleted.
```

Comment:
Could you please update this sentence to 'Models and predictions of the worst-performing models will be deleted then'.